### DISCUSSION TITLE
Optimize images with Pingo/Imagemagick

### DISCUSSION THREAD

**Discussion by AerialAtom** [Difegue/LANraragi#1102] [Ideas]
Would it be possible to add pingo/pinga to optimize the images in the archives. It can be when it's finished downloading them or by opening the archive. It would be helpful if it had options like pinga or maybe imgdanke.

**Additional context**
https://css-ig.net/
https://github.com/DrWhoCares/imgdanke
![image](https://github.com/user-attachments/assets/1e4d833f-286b-40f9-9ed7-45bfea19d1c3)
![109585884-e988bd80-7ac9-11eb-9a28-3767206ac663](https://github.com/user-attachments/assets/30429522-f8cf-4ae2-89f3-54caea8a7346)

**Comment by Difegue** [comment]
I'm not seeing any source code or usable library for this, only a windows executable.  

Unless I missed something I don't see how I can integrate this - There's also already an existing option within the server to compress images after unarchiving them to save on bandwidth.
